src/catalog/src/remote/manager.rs

```diff
@@ -430,23 +425,30 @@ impl RemoteCatalogManager {
 async fn iter_remote_schemas<'a>(
     backend: &'a KvBackendRef,
     catalog_name: &'a str,
-) -> Pin<Box<dyn Stream<Item = Result<SchemaKey>> + Send + 'a>> {
+) -> Result<Vec<SchemaKey>> {
```

Comment:
Because we are using the Metasrv as one kind of KvBackend, too. In that case, we have to change the grpc service to server side streaming to fully make the range API "stream"ed. Which seems an overkill for this. In fact, in the original version of MetaKvBackend, it created a stream locally after received the whole data from Metasrv. Besides, the range response has a "more" field, to indicate the client to request more range request thereafter. This can be some sort of flow control.